### ISSUE TITLE
How to add description for workflow? [Docs] 

### ISSUE THREAD

**Issue by y8gao** [flyteorg/flyte#2847]
### Description

I checked the webpages in docs, but not found any document that describes how to add description of workflow in FlyteUI.
And in Flyte UI, Description section always says "this workflow has no description".
![image](https://user-images.githubusercontent.com/5154290/188398062-c388280f-a8ed-4541-9ea1-517013df3c7e.png)

So my question is: how to add this description or where can I find the related contents?

### Are you sure this issue hasn't been raised already?

- [X] Yes

### Have you read the Code of Conduct?

- [X] Yes

**Comment by pingsutw** [comment]
You could update the description by using [flytectl](https://docs.flyte.org/projects/flytectl/en/latest/gen/flytectl_update_workflow-meta.html#).
```
flytectl update workflow-meta -p flytesnacks -d development example_test.wf --description "Mergesort workflow example"
```

**Comment by wild-endeavor** [comment]
You shouldn't have to use flytectl though... there's ongoing work to bring in user comments.  In the hopefully near future, the registration process will just pick up comments from the code for tasks, workflows, and input parameters (and output parameters? @pingsutw )

**Comment by wild-endeavor** [comment]
Adding pingsutw to this ticket, he's working on it next week.  we'll post updates here when we have them.  thank

**Comment by y8gao** [comment]
> 

Good to know this updates. Thanks, and I will try this new version after it is ready.  Thanks for your comments.

**Comment by eapolinario** [comment]
This has been implemented in flytekit 1.3.0.
